pipeline/scripts/4/2/tier-2/test-deploy_okr_bz_1994827.sh

```diff
@@ -0,0 +1,49 @@
+#! /bin/sh
+echo "Begin testing of Red Hat Ceph RPM based upgrade."
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-7"
+rhbuild="4.2"
+test_suite="suites/nautilus/upgrades/tier-2_okr_bz_1994827.yaml"
+test_conf="conf/nautilus/upgrades/tier-2_okr_BZ_1994827.yaml"
+test_inventory="conf/inventory/rhel-7.9-server-x86_64.yaml"
+return_code=0
+
+# Process the CLI arguments for IBM-C environment
+CLI_ARGS=$@
+cloud="ibmc"
+if [ -z "${CLI_ARGS##*$cloud*}" ] ; then
+    test_inventory="conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
```

Comment:
Could you use the soft link which points to the latest one? That is, `ibm-vpc-rhel-7-latest.yaml`